Solutions/Marain.Workflow.Storage.Sql/Marain/Workflows/Storage/SqlWorkflowStore.cs

```diff
@@ -51,7 +51,7 @@ public Task UpsertWorkflowAsync(Workflow workflow, string partitionKey = null)
 
         private async Task<Workflow> GetWorkflowCoreAsync(string workflowId)
         {
-            using SqlConnection connection = this.connectionFactory();
+            using SqlConnection connection = await this.connectionFactory().ConfigureAwait(false);
```

Comment:
Since you're using `using` here, the `connection.Close()` call on line 73 seems redundant. Ditto for the reader.

Reply:
True. But opinion is divided on whether it is "better documenting the golden path doneness" to explicitly Close(). I don't mind either way.